src/ml_gym/frontend/dashboard/src/routes/settings.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import {useAppSelector} from "../app/hooks"
+
+type SettingsProps = {
+};
+
+
+const Settings: React.FC<SettingsProps> = ({ }) => {
+    return (
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=9966992)
```suggestion
  return (
```